examples/actors/mutate_remote_state.py

```diff
@@ -0,0 +1,64 @@
+from itertools import cycle
+from pprint import pformat
+from dataclasses import dataclass, field
+
+import trio
+import tractor
+
+
+@dataclass
+class MyProcessStateThing:
```

Comment:
Instead of this what are you after?

A function that creates some object and then makes that object mutateable from another inbound message?

Reply:
Maybe you want something like `ray`s "actors" (which i would argue aren't really "actor model" actors):
https://docs.ray.io/en/latest/actors.html

We can also accomplish this but it will require  a slight bit more machinery.